### PR TITLE
Removed Mortar and Flow

### PR DESCRIPTION
**Pull request by cristhianescobar**
- [ ] @ayvazj  
- [ ] @christophior

### CODE REVIEW

File: app/templates/_build.gradle [hunk 1]
```diff
@@ -1,14 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 ext {
-    minSdkVersion = <%= androidMinSdkVersion %>
+    minSdkVersion = 19
```
Comment (ayvazj) [inline app/templates/_build.gradle:4]: intentional?

File: app/templates/app/_build.gradle [hunk 2]
```diff
@@ -20,8 +20,8 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.neenbedankt.android-apt'
 
 android {
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdkVersion 23
```
Comment (ayvazj) [inline app/templates/app/_build.gradle:23]: intentional?

File: app/templates/_build.gradle [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
     targetSdkVersion = <%= androidTargetSdkVersion %>
     compileSdkVersion = 23
-    buildToolsVersion = "23.0.2"
-    supportLibraryVersion = '23.3.0'
+    buildToolsVersion = "23.0.3"
+    supportLibraryVersion = '23.1.1'
```
Comment (ayvazj) [inline app/templates/_build.gradle]: 23.1.1 downgrade?

File: app/templates/_build.gradle (hunk 1 above)
Comment (cristhianescobar) [inline app/templates/_build.gradle:4]: Yeah, for now. I will bring back on the next PR when I get the sign in stuff working properly

File: app/templates/app/_build.gradle (hunk 2 above)
Comment (cristhianescobar) [inline app/templates/app/_build.gradle:23]: Same

File: app/templates/app/src/main/java/_Application.java [hunk 4]
```diff
@@ -1,42 +1,32 @@
...
 import timber.log.Timber;
 
 public class Application extends android.app.Application {
 
-    private MortarScope mortarScope;
-
     @Override
     public Object getSystemService(String name) {
-        return mortarScope.hasService(name) ? mortarScope.getService(name) : super.getSystemService(name);
+        return super.getSystemService(name);
```
Comment (ayvazj) [inline app/templates/app/src/main/java/_Application.java]: can we get ride of this @Override all together?

File: app/templates/_build.gradle (hunk 3 above)
Comment (cristhianescobar) [inline app/templates/_build.gradle]: Fixed

File: app/templates/app/src/main/java/screen/main/_MainActivity.java [hunk 5]
```diff
@@ -1,408 +0,0 @@
...
-    }
-
-    private class UpSelectionHandler implements MenuItemSelectionHandler {
-        @Override
-        public boolean execute() {
-            return containerAsHandlesBack.onBackPressed();
-        }
-    }
-
-    private class SwitchEnvironmentSelectionHandler implements MenuItemSelectionHandler {
```
Comment (ayvazj) [inline app/templates/app/src/main/java/screen/main/_MainActivity.java:377]: Consider carrying this forward.  Let's you switch between test and production instances.

**Comment by ayvazj** [comment]
Need to update BaseTest

File: app/templates/app/src/main/java/_Application.java (hunk 4 above)
Comment (cristhianescobar) [inline app/templates/app/src/main/java/_Application.java]: Fixed

File: app/templates/app/src/main/java/screen/main/_MainActivity.java (hunk 5 above)
Comment (cristhianescobar) [inline app/templates/app/src/main/java/screen/main/_MainActivity.java:377]: How about we enable Dev Options (where we could switch environment) somewhere else other than the toolbar? Maybe we can even trigger Dev Options through a shake?

Comment (cristhianescobar) [inline app/templates/app/src/main/java/screen/main/_MainActivity.java:377]: Wait a second we already have this capability through gradle

**Comment by christophior** [comment]
You can't merge when I haven't given this a 👍

**Pull request merged**